### PR TITLE
Fix links in the UI

### PR DESCRIPTION
**Pull request by moyodiallo**
This PR fix the issue #26

The `admin-service-uri` parameter could have or not a slash at the end.

### CODE REVIEW

**Review by MisterDA — APPROVED** [review]
LGTM

File: web-ui/main.ml [hunk 1]
```diff
@@ -80,6 +80,10 @@ module Graphql_cohttp_lwt = Graphql_cohttp.Make (Graphql_lwt.Schema) (Cohttp_lwt
 
 let main port backend_uri admin_service_uri docroot prometheus_config =
   Lwt_main.run begin
+    let admin_service_uri = match String.cut ~sep:"/" ~rev:true (String.trim admin_service_uri) with
+    | Some (uri,"") -> uri
+    | _ -> admin_service_uri
+    in
     let vat = Capnp_rpc_unix.client_only_vat () in
```
Comment (tmcgilchrist) [inline web-ui/main.ml:89]: Use the `Uri` module to parse this into a Uri.t (to validate it) and then print it out from a Uri.t

Comment (moyodiallo) [reply web-ui/main.ml:89]: > Use the `Uri` module to parse this into a Uri.t (to validate it) and then print it out from a Uri.t

I used `Uri` module, the slash at the end still stays.

Comment (tmcgilchrist) [reply web-ui/main.ml:89]: Never mind then, go with what is here.

**Pull request merged**